client/components/add-beer-view/add-beer-view.jsx

```diff
@@ -72,62 +76,70 @@ class AddBeerView extends React.Component {
     const { bind } = this;
     const { bars, beerTypes } = this.props;
     const {
-      bar, beerType, volume, abv, description, isSubmitting,
+      bar, beerType, volume, abv, price, description, isSubmitting,
     } = this.state;
 
     const isValid = (
       bar !== DEFAULT_BAR
       && beerType !== DEFAULT_BEER_TYPE
-      && volume !== DEFAULT_VOLUME);
+      && volume !== DEFAULT_VOLUME
+      && this.state.abv.length > 0
+      && this.state.price.length > 0);
```

Comment:
the `price` and `avb` are already available in the scope, so use those variables

Reply:
Alright